Dockerfile

```diff
@@ -1,16 +1,13 @@
-FROM kong:0.11
+FROM kong:2.0.4
```

Comment:
It's possible to use alpine version?

Reply:
I think It's possible use kong:2.0.4-alpine, I will test to be sure.

Reply:
I tested and It's ok :)